zerver/openapi/zulip.yaml

```diff
@@ -20448,12 +20448,15 @@ paths:
 
                         Administrators can always administer a channel.
```

Comment:
Should this now say "Organization administrators can administer every channel as though they were in this group without being explicitly listed here."